### DISCUSSION TITLE
In primeng 17 if the value set in p-calendar and [maxDate] are same then the value is not showing selected

### DISCUSSION THREAD

**Discussion by jghosh-argusoft** [primefaces/community#2406] [PrimeNG]
Here is the demo i made in stackblitz , https://stackblitz.com/edit/nb7yxf?file=src%2Fapp%2Fcalendar-minmax-demo.ts

I tried to bind the date value in p-calendar it works for every date except it matches with the date set in [maxDate]. For example if the [maxDate] is set to 31st december and if the date value in the p-calendar is 31st december then the value is not showing in the p-calendar (priemng 17).

**Comment by pierocox09** [comment]
me pasa lo mismo
